### ISSUE TITLE
`python_qemu fuzzer.py` undefined reference to `run()`

### ISSUE THREAD

**Issue by psparc82** [AFLplusplus/LibAFL#2002]
When trying to run `fuzzer.py` from the `python_qemu` example folder the script fails with an unknown reference to the emulator `emu.run()` function. To get this far I had to change line 9 from:

```python3
emu = qemu.Emulator(['qemu-x86_64', BINARY_PATH], [])
```

to:

```python3
emu = qemu.Qemu(['qemu-x86_64', BINARY_PATH], [])
```

This lets me call other emulator functions such as `emu.set_breakpoint()` just fine but not `emu.run()`.

Line 18 is the issue:

```python3
emu.run()
```

There is no `run()` function on the `emu` object. I think quite a few things have changed/moved around with the emulator and have broken the example `fuzzer.py`. 

Any help?

Thanks!

**Comment by tokatoka** [comment]
nobody is actively maintaining python bindings now. i recommend not using it..

**Comment by psparc82** [comment]
> nobody is actively maintaining python bindings now. i recommend not using it..

ah, I thought that may be the case. Do you know of any up to date, solid examples of using Qemu with `libafl sugar` that I can learn from?

**Comment by tokatoka** [comment]
sugar is not really maintained as well.
but at least i think it's not broken cuz it's not written in python3.
you can just look at some examples in fuzzers/ that don't have "sugar" or "python" in its name.

**Comment by tokatoka** [comment]
also fuzzbench_qemu is good as these fuzzbench fuzzer are single-process and easy to debug what is happening

**Comment by domenukk** [comment]
@psparc82 We decided it may be worthwhile to keep the bindings for qemu around, as scriptable emulation can be pretty powerful. Help fixing these issues is always appreciated :)

**Comment by domenukk** [comment]
Fixed in #2030 @psparc82 , thanks for reporting
